reactor-netty-http/src/main/java/reactor/netty/http/server/HttpServer.java

```diff
@@ -290,6 +290,21 @@ public final HttpServer httpRequestDecoder(Function<HttpRequestDecoderSpec, Http
 		return dup;
 	}
 
+	/**
+	 * Decorate the configured I/O handler.
+	 * See {@link #handle(BiFunction)}.
+	 *
+	 * @param mapHandle A function to decorate the configured I/O handler
+	 * @return a new {@link HttpServer}
+	 */
+	public final HttpServer mapHandle(Function<? super Mono<Void>, ? extends Mono<Void>> mapHandle) {
+		Objects.requireNonNull(mapHandle, "mapHandle");
+		HttpServer dup = duplicate();
+		Function<? super Mono<Void>, ? extends Mono<Void>> current = configuration().mapHandle;
+		dup.configuration().mapHandle = current == null ? mapHandle : current.andThen(mapHandle);
```

Comment:
do you foresee a case where one would want to remove mapHandle `Function` (or part(s) of it)? after it has been set?

Reply:
For the moment no, but if it is needed we can add an API.
I will update the javadoc for this that is adds this function to the existing functions (if any)
wdyt?

Reply:
ah yeah that's probably worth mentioning in the javadoc